authui/src/authflowv2/components/recovery-code-list.css

```diff
@@ -0,0 +1,63 @@
+@layer components {
+  :root {
+    --recovery-code-list__bg-color: var(--color-grey-100);
+    --recovery-code-list__px: 1em;
+    --recovery-code-list__py: 1em;
+    --recovery-code-list__border-radius: 0.9375em;
+    --recovery-code-list__font-size: 0.75em;
+    --recovery-code-list__font-weight: 500;
+    --recovery-code-list__color: var(--color-grey-700);
+    --recovery-code-list__line-height: 1rem;
+    --recovery-code-list__letter-spacing: 0.025rem;
+
+    --recovery-code-list__btn-bg-color: var(--color-grey-700);
+    --recovery-code-list__btn-border-radius: 6.25em;
+    --recovery-code-list__btn-px: 1.5em;
+    --recovery-code-list__btn-py: calc(
+      var(--recovery-code-list__btn-font-size) * 13 / 14
+    );
```

Comment:
Can we use em? And also please document how you constructed this formula.

Reply:
As mentioned in https://github.com/authgear/authgear-server/pull/3663/files#r1462773268 I think recovery-code__btn can be removed